Kickstarter-iOS/Views/Controllers/PledgeDescriptionViewController.swift

```diff
@@ -168,6 +304,15 @@ private let descriptionStackViewStyle: StackViewStyle = { (stackView: UIStackVie
   stackView
     |> \.axis .~ NSLayoutConstraint.Axis.vertical
     |> \.distribution .~ UIStackView.Distribution.fill
+    |> \.spacing .~ Layout.Description.spacing
```

Comment:
Is there a reason this spacing is stored in an enum? Additionally, should we be using `Style.grid` units for this spacing?

Reply:
Good q's - I think in this view initially referred to the spacing in two places so I moved it to a constant. Also in that view it was just `10`. We can't really get to that on the grid spacing but I could make it .grid(2).

Reply:
It's a little unclear from the naming what this spacing is for - let's remove it if it's only used here?